Source/RageCommunity.Library/Extensions/VectorExtensions.cs

```diff
@@ -62,5 +62,35 @@ public static Vector3 GetOffset(this Vector3 from, float heading, Vector3 offset
 
             return new Vector3(from.X + resultX, from.Y + resultY, from.Z + offset.Z);
         }
+        /// <summary>
+        /// get a heading between two position
+        /// </summary>
+        /// <returns>return the heading</returns>
+        /// <remarks>
+        /// Source: <a href="http://answers.unity.com/answers/697834/view.html"></a>
+        /// </remarks>
+        public static float GetHeadingBetweenTwoVector(this Vector3 from, Vector3 to)
```

Comment:
I suggest to name this "GetHeadingToVector" because it is an extension method and will be called like: 
myVector.GetHeadingDifference(otherVector)
IMHO this is better to read than:
myVector.GetHeadingBetweenTwoVector(otherVector)

Reply:
maybe i just change it to `GetHeadingTowardsVector`?

Reply:
Yes, that would also be a good name :)